custom_components/smartthinq_sensors/sensor.py

```diff
@@ -187,14 +187,14 @@ class ThinQSensorEntityDescription(SensorEntityDescription):
     ),
     ThinQSensorEntityDescription(
         key=ATTR_REMAIN_TIME,
-        name="Remain time",
+        name="Remaining time",
```

Comment:
This will generate same issue as for `tube_clean` but I suppose that this sensor is more used. Issue opened will grow exponentially